app/views/tvs/index.html.erb

```diff
@@ -1,42 +1,47 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <%= render 'layouts/components/headings/one', text: 'Teaching Vacancies' %>
-    <p class="govuk-body">Below are the 50 most recent teaching vacancies. View these and more on the
-      <a class="govuk-link" href="https://teaching-vacancies.service.gov.uk/">Teaching Vacancies Service</a></p>
-    <%= render 'layouts/components/headings/two', text: 'Recent job postings' %>
-    <% @response.each do |job| %>
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-          <span class="govuk-caption-l"><%= "#{job['hiringOrganization']['name']}, #{job['jobLocation']['address']['addressLocality']}, #{job['jobLocation']['address']['addressRegion']}" %></span>
-          <h2 class="govuk-heading-l"><%= job['title'] %></h2>
-          <dl class="govuk-summary-list govuk-summary-list--no-border">
-            <div class="govuk-summary-list__row">
-              <dt class="govuk-summary-list__key">
-                Salary
-              </dt>
-              <dd class="govuk-summary-list__value">
-                <% if job['baseSalary']['value']['minValue'] && job['baseSalary']['value']['maxValue'] %>
-                  <%= "£#{job['baseSalary']['value']['minValue']} to £#{job['baseSalary']['value']['maxValue']} per #{job['baseSalary']['value']['unitText'] == 'YEAR' ? 'year' : '???'}" %>
-                <% else %>
-                  <%= "£#{job['baseSalary']['value']['value']} per #{job['baseSalary']['value']['unitText'] == 'YEAR' ? 'year' : '???'}" %>
-                <% end %>
-              </dd>
+<div class="govuk-width-container">
+  <%= render 'layouts/components/images/hero', image_src: asset_url('home-hero-lady.jpg') %>
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <%= render 'layouts/components/headings/one', text: 'Teaching Vacancies' %>
+        <p class="govuk-body">Below are the 50 most recent teaching vacancies. View these and more on the
```

Comment:
50 is quite a lot. We display 10 on a page.

Reply:
You're not wrong - but I can address this in a future PR :eyes: probably worth cutting down the number, it's more just a proof of concept at the moment